app/components/application-wrapper.js

```diff
@@ -64,6 +67,14 @@ export default Component.extend({
 		return `${vertical}-vertical`;
 	}),
 
+	fandomAppSmartBannerObserver: observer('isFandomAppSmartBannerVisible', () => {
+		if (true || inGroup('FEATURED_VIDEO_VIEWABILITY_VARIANTS', 'RENDER_ORDER')) {
+			scheduleOnce('afterRender', () => {
```

Comment:
recently we've learned that this afterRender might not happen when you think it'll happen
but for a test it should be OK